addons/mail/static/src/core/common/store_service.js

```diff
@@ -386,30 +387,38 @@ export class Store extends BaseStore {
             ev.preventDefault();
             this.openChat({ partnerId: id });
             return true;
-        } else if (link.classList.contains("o_message_redirect_transformed") && id) {
+        } else if (link.classList.contains("o_message_redirect")) {
             const message = this["mail.message"].get(id);
             const targetThread = message?.thread;
+            const sendError = () =>
```

Comment:
Send error feels generic (and is not an error?)

```suggestion
            const sendUnavailableNotification = () =>
```
or 
```suggestion
            const sendUnavailableWarning = () =>
```